packages/apps/src/overlays/Connecting.tsx

```diff
@@ -17,59 +17,57 @@ type Props = I18nProps & ApiProps;
 // @ts-ignore
 const isFirefox = typeof InstallTrigger !== 'undefined';
 
-class Connecting extends React.PureComponent<Props> {
-  public render (): React.ReactNode {
-    return this.renderExtension() || this.renderConnecting();
+function renderExtension ({ className, isWaitingInjected, t }: Props): React.ReactNode {
+  if (!isWaitingInjected) {
+    return null;
   }
 
-  private renderExtension (): React.ReactNode {
-    const { className, isWaitingInjected, t } = this.props;
-
-    if (!isWaitingInjected) {
-      return null;
-    }
+  return (
+    <BaseOverlay
+      className={className}
+      icon='puzzle'
+    >
+      <div>{t('Waiting for authorization from the extension. Please open the installed extension and approve or reject access.')}</div>
+    </BaseOverlay>
+  );
+}
 
-    return (
-      <BaseOverlay
-        className={className}
-        icon='puzzle'
-      >
-        <div>{t('Waiting for authorization from the extension. Please open the installed extension and approve or reject access.')}</div>
-      </BaseOverlay>
-    );
+function renderConnecting ({ className, isApiConnected, t }: Props): React.ReactNode {
+  if (isApiConnected) {
+    return null;
   }
 
-  private renderConnecting (): React.ReactNode {
-    const { className, isApiConnected, t } = this.props;
-
-    if (isApiConnected) {
-      return null;
-    }
+  const wsUrl = settings.apiUrl;
+  const isWs = wsUrl.indexOf('ws://') === 0;
+  const isWsRemote = wsUrl.indexOf('127.0.0.1') === -1;
+  const isHttps = window.location.protocol.indexOf('https:') === 0;
 
-    const wsUrl = settings.apiUrl;
-    const isWs = wsUrl.indexOf('ws://') === 0;
-    const isWsRemote = wsUrl.indexOf('127.0.0.1') === -1;
-    const isHttps = window.location.protocol.indexOf('https:') === 0;
+  return (
+    <BaseOverlay
+      className={className}
+      icon='globe'
+    >
+      <div>{t('You are not connected to a node. Ensure that your node is running and that the Websocket endpoint is reachable.')}</div>
+      {
+        isFirefox && isWs
+          ? <div>{t('With the Firefox browser connecting to insecure WebSockets ({{wsUrl}}) will fail due to the browser not allowing localhost access from a secure site.', { replace: { wsUrl } })}</div>
+          : undefined
```

Comment:
Completely throw-away comment - I always wonder about these where we mix null and undefined. Doesn’t make a difference either way, but we are inconsistent. 

Don’t do anything, just saw it